src/components/about-us/utils/lottie.js

```diff
@@ -29,19 +29,21 @@ export default class Lottie extends React.Component {
     this.registerEvents(eventListeners)
   }
 
-  getSnapshotBeforeUpdate(prevProps/*, prevState*/) {
+  shouldComponentUpdate(nextProps /* , nextState */) {
     /* Recreate the animation handle if the data is changed */
-    if (prevProps.options.animationData !== this.props.options.animationData) {
-      this.deRegisterEvents(prevProps.eventListeners)
+    if (this.options.animationData !== nextProps.options.animationData) {
+      this.deRegisterEvents(this.props.eventListeners)
       this.destroy()
-      this.options = { ...this.options, ...this.props.options }
-      this.anim = lottie.loadAnimation(this.options)
-      this.registerEvents(this.props.eventListeners)
+      this.registerEvents(nextProps.eventListeners)
+      return true
     }
-    return null 
+    return false
   }
 
   componentDidUpdate() {
+    this.options = { ...this.options, ...this.props.options }
```

Comment:
Updated.

In this line, `this.options` and `this.props.option` need to be merged together as the updated `this.options` since `this.options` contains local variables which are not existed in `this.props.options`.